packages/terra-pills/src/PillList.jsx

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useRef, useState } from 'react';
+import { KEY_SPACE, KEY_RETURN } from 'keycode-js';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+import styles from './Pill.module.scss';
+import Pill from './Pill';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * The string that labels the collection of pills, used in cased where the text label is not visible on the screen and required for minimum accessibility standards.
+   * Providing this prop adds the `aria-label` attribute to the Pill List container element. (Required)
+   */
+  ariaLabel: PropTypes.string.isRequired,
+  /**
+   * If a visible text label is used with the collection of pills, provide a string of the ID for the html element containing the visible text label.
+   * Supplying the 'ariaLabelledBy' prop will override the 'ariaLabel' prop and adds the `aria-labelledby` attribute instead to the Pill List container element.
+   *
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue) For best practices, ensure the visible text in the html element `id` provided to the Pill List
+   * `ariaLabelledby`prop matches the text provided to the `ariaLabel` prop, for consistency in the case of fallback or errors.
+   */
+  ariaLabelledBy: PropTypes.string,
+  /**
+   * If additional visible information text is used, provide a string containing the IDs for html elements that help describe the intent of the pills and the Pill List container.
+   */
+  ariaDescribedBy: PropTypes.string,
+  /**
+   * The content to be shown in the pills container. Should only contain the sub-component pills.
+   */
+  children: PropTypes.node,
+  /**
+   * Determines if the Pill List is rolled up or not.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * Callback function triggered on click/key press of the roll-up pill
+   */
+  rollUpPillOnTrigger: PropTypes.func,
```

Comment:
Callback props are typically prefixed with "onSelect". I'd expect this to be something like "onSelectRollUp"

Reply:
Will change it to 'onSelectRollUp'